rules/library.bzl

```diff
@@ -192,12 +192,12 @@ def _prepend(list, other):
 
 def _prepend_copts(copts_struct, objc_copts, cc_copts, swift_copts, linkopts, ibtool_copts, momc_copts, mapc_copts):
     _prepend(objc_copts, copts_struct.objc_copts)
-    _prepend(copts_struct.cc_copts, cc_copts)
-    _prepend(copts_struct.swift_copts, swift_copts)
-    _prepend(copts_struct.linkopts, linkopts)
-    _prepend(copts_struct.ibtool_copts, ibtool_copts)
-    _prepend(copts_struct.momc_copts, momc_copts)
-    _prepend(copts_struct.mapc_copts, mapc_copts)
+    _prepend(cc_copts, copts_struct.cc_copts)
```

Comment:
... bad sam, this has been broken for a while?

Reply:
It this just a matter of precedence?

Reply:
No -- we were mutating the wrong variable, meaning swift copts from xcconfigs weren't being applied to `swift_library` targets.  the new test framework shows that this is fixed now